### PR TITLE
fix: making customizations in line with Paragon

### PR DESCRIPTION
**Pull request by kiram15**
PR needed after conversations with Jenn/Amy and Adrienne on making our components more inline with Paragon standards and removing as much css customization as possible.

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -19,22 +19,11 @@
...
-.alacarte-catalog.badge  {
+.padded-catalog.badge  {
     margin: 4px;
-    padding: 6px;
-    max-width: 20vw;
-    color: #ffffff;
-    font-weight: 300;
-    background-color: #202020;
-  }
+}
```
Comment (edx-abolger) [inline src/index.scss:25]: This looks great, very theme-compliant!!!

**Review by edx-abolger — APPROVED** [review]
This looks great, love the $theme compliance and labels!

**Comment by edx-abolger** [comment]
@kiram15 don't forget to attach screenshots to this PR before merging, otherwise GTG

File: src/components/catalogPage/CatalogPage.messages.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { defineMessages } from '@edx/frontend-platform/i18n';
 const messages = defineMessages({
   'catalogPage.hero.text': {
     id: 'catalogPage.hero.text',
-    defaultMessage: 'World-class learning for every organization ',
+    defaultMessage: 'World-class learning \n for every organization ',
```
Comment (alex-sheehan-edx) [inline src/components/catalogPage/CatalogPage.messages.js:6]: 👍

**Review by alex-sheehan-edx — APPROVED** [review]
This is great! My only (not mandatory) ask is for screen shots of before/after in the PR when doing style/formatting updates 😇. It makes it easier for non-coding people/people who don't have things set up locally to confirm changes.

**Pull request merged**